src/main/scala/wdl4s/Scope.scala

```diff
@@ -181,8 +181,8 @@ trait Scope {
     * @param knownInputs All known values of FQNs
     * @param wdlFunctions Implementation of WDL functions for expression evaluation
     * @param shards For resolving specific shards of scatter blocks
-    * @return String => WdlValue lookup function rooted at `scope`
```

Comment:
Good catch. Overly aggressive removal. The one on the next line was causing bad scaladoc rendering [due to](https://wiki.scala-lang.org/display/SW/Syntax):
> Headings: =heading= defines a heading. Lower-level heading are obtained by using more = signs, like ===sub-heading===. A heading must be define on its own line.

producing:
```bash
[warn] /Users/kshakir/src/wdl4s/src/main/scala/wdl4s/Scope.scala:176: unbalanced or unclosed heading
[warn]   /**
[warn]   ^
```

Apparently `@return` doesn't render the same way as `@throws`, and the former rocket shows up just fine in the rendered html.

Will put it back in a squash.